.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Report
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
I would replace the section by title in place of bold text. It will avoid having title section and the text on the same line once rendered.

```suggestion
#### Describe the bug
```

Reply:
Done as desired